src/content/docs/it/tutorial/1-setup/2.mdx

```diff
@@ -0,0 +1,147 @@
+---
+type: tutorial
+title: Crea il tuo primo progetto Astro
+description: |-
+  Tutorial: Crea il tuo primo blog Astro —
+  Crea un nuovo progetto per il tutorial e preparati a programmare
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+<PreCheck>
+  - Esegui l'installazione guidata `create astro` per creare un nuovo progetto Astro
+  - Avvia il server Astro in modalità sviluppo (dev)
+  - Visualizza una preview in tempo reale del tuo sito nel tuo browser
+</PreCheck>
+
+## Esegui l'installazione guidata di Astro
+
+Il metodo preferito per creare un nuovo sito Astro è tramite la nostra installazione guidata `create astro`.
+
+1. Nel promt dei comandi del tuo terminale, esegui il comando seguente usando il tuo package manager preferito:
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      # crea un nuovo progetto con npm
+      npm create astro@latest
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      # crea un nuovo progetto con pnpm
+      pnpm create astro@latest
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      # crea un nuovo progetto con yarn
+      yarn create astro
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+2. Conferma `y` per installare `create-astro`
+3. Quando il prompt chiede, "Where would you like to create your new project?" inserisci il nome della cartella per creare una nuova directory per il tuo progetto, per esempio `./tutorial`
+
+    :::note
+    Un nuovo progetto Astro può solo essere creato in una cartella completamente vuota, quindi scegli un nome per la tua cartella che non esista già!
+    :::
+
+4. Vedrai una breve lista di template con cui iniziare. Usa le freccie (sù e giù) per navigare al template "Empty", e poi premi invio (enter) per inviare la tua scelta.
+
+5. Quando il prompt chiede, "Would you like to install dependencies?" inserisci `y`.
+
+6. Quando il prompt chiede se pianifichi di utilizzare TypeScript, inserisci `n`.
+
+7. Quando il prompt chiede, "Would you like to initialize a new git repository?" inserisci `y`.
```

Comment:
```suggestion
7. Quando il prompt chiede "Would you like to initialize a new git repository?", inserisci `y`.
```
I would not use the comma after "chiede". Rather I would use a comma after the direct question, like in this example:
```
“Ma tu cosa vai cercando?”, gli ha detto spingendolo fuori - Antonio Tabucchi.
```

I know, in the original sentence commas are used the same way, but I would prefer a more correct usage if it's fine with you.

Reply:
I also think it's preferable to use them as you say. I am not so good at punctuation so I didn't know if I was wrong or not ^.^ . Anyway, thanks for the suggestion